docs/FAQ.md

```diff
@@ -22,3 +22,11 @@ Yes! In our effort to create the highest-quality C++ libraries, we've establishe
 
 * Our [Example Project](https://github.com/beman-project/example) demonstrates a useful structure for organizing libraries.
 * Our [CI Project](https://github.com/beman-project/ci) utilizes Github Workflows to automate building, testing, and releasing libraries.
+
+## Will libraries stay in Beman forever?
+
+No! The current concept is that libraries will be gradually deprecated as official standard library implementations roll out. We expect this to coorespond to two standards cycles. So a library accepted into C++26 would be removed by C++32. Users can, of course, still depend on the original repo for the implementation if they don't want to change to ~std::~ for seme reason.
+
+## Is there are review process like Boost?
```

Comment:
Maybe mention the ISO process _is_ the review process?

Of course, more informal peer review will generally be more productive, etc.

Reply:
yes -- this is a good idea.